### PR TITLE
Revise schemas

### PR DESCRIPTION
**Pull request by milofultz**
Adapted my schemas to match the provided CSVs

### CODE REVIEW

File: server/nosql-test.js [hunk 1]
```diff
@@ -1,47 +1,67 @@
-const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost:27017/atelier', {
+const mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost:27017/atelier", {
   useNewUrlParser: true,
-  useUnifiedTopology: true
+  useUnifiedTopology: true,
```
Comment (NylonEric) [inline server/nosql-test.js:4]: one change; one git line addition,

File: schemas/postgres.sql [hunk 2]
```diff
@@ -53,9 +54,8 @@ CREATE TABLE IF NOT EXISTS characteristic_ratings (
...
 CREATE TABLE IF NOT EXISTS characteristics (
-  characteristic_id INT GENERATED ALWAYS AS IDENTITY,
+  id INT GENERATED ALWAYS AS IDENTITY,
+  product_id INT,
   value TEXT,
-  PRIMARY KEY(characteristic_id)
+  PRIMARY KEY(id)
 );
-
-SELECT * FROM characteristics;
```
Comment (NylonEric) [inline schemas/postgres.sql:61]: Is this statement used in any server method/script or is it just to be called from the command line once for creation?

**Review by NylonEric — APPROVED** [review]
This looks good. I know you have decided to go with a nosql database for your service so my questions may be more relevant to the actual architecture that you are using.

File: schemas/postgres.sql (hunk 2 above)
Comment (milofultz) [reply schemas/postgres.sql:61]: I was just using it for testing and forgot to remove it

**Pull request merged**